### ISSUE TITLE
Find method for RI prediction for different RI types on NIST and Wiley

### ISSUE THREAD

**Issue by hechth** [RECETOX/RIAssigner#104]
We need to train our own model -> first part would be to start with an scipy SVM using computed fingerprints from matchms and then train a model.

Find out how scipy can store a model and how it can be loaded. this should be first implemented in a jupyter notebook  and then if the accuracy is within 20 points, we can put this into RIAssigner as a function and implement 2 new Galaxy tools for this.

Details on the Scipy implementation of an SVM are available here: https://scikit-learn.org/stable/modules/svm.html#implementation-details

Accuracy for training set should be within 20 points

Related papers:
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3163042/
https://www.sciencedirect.com/science/article/abs/pii/S0169743920301416
https://www.sciencedirect.com/science/article/pii/S0021967323004028
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8430916/
